### PR TITLE
Add more content to Python-Resources page

### PR DESCRIPTION
**Pull request by richyvk**
Added content to Tutorials and Challenges, and created new Blogs, Courses (i.e. MOOCS), Podcasts, and Hosting sections.

### CODE REVIEW

**Comment by raisedadead** [comment]
Thanks @richyvk for the PR. Looks good to me.

Please [Squash the commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) after go ahead from @alayek

File: Python-Resources.md [hunk 1]
```diff
@@ -37,3 +47,26 @@
...
+
+## Blogs
+
+* [Real Python Blog](https://realpython.com/blog/)
+* [Python Tips](http://pythontips.com/)
+* [Schooling The N00b](https://schoolingthenoob.wordpress.com/)
+
+## Podcasts
+
+* [Podcast.__init__](http://pythonpodcast.com/)
```
Comment (koustuvsinha) [inline Python-Resources.md]: The markdown rendered it just into a bold Podcast.__init__, better write it as `Podcast.\__init__` -> Podcast.\__init__

Comment (raisedadead) [inline Python-Resources.md]: Ahh right, good catch @koustuvsinha

**Comment by alayek** [comment]
@richyvk thanks for the PR. It is great to see such abundance of resources to help people learn Python.

However, if you notice original commit by @koustuvsinha, we have not included many of this. The reason is not that we don't think they are great resources. They are indeed, as many who have followed them and got benefited would attest to.

The reason is licensing conflict. FCC sells T-shirts to raise fund, and charges businesses to post on our job boards.

In a way, we are not completely non-profit, at least in the legal sense of the word. Some of the links added here also follow a similar business model, whereas they make profit; but not directly charging every customer.

I would not be much mistaken, if a court of law views them as our competitor. And linking to their resources in our official wiki would amount to some sort of business law violation (kindly contact an attorney in your local area to understand these in much detail), without their consent.

We have only linked to resources that are free, open-sourced; and licensed in a way that others can refer to it without their permission.

Just trying to save us a CDMA takedown notice.

Disclaimer: I am not a legal expert. I may have been completely wrong in my above analysis. You should reach out to an attorney of your choice to get more legal advice on this.

**Comment by richyvk** [comment]
@alayek Thanks for your feedback. I confess, I didn't really think about the legality of linking to these resources. I am in fact a law librarian in my day job, but not a lawyer. Nevertheless I realise the law can be very tricky and FCC has to be careful. I don't think I have the time or knowledge to determine which of the resources I linked to are OK legally, so maybe it's best not to include any of them at this stage then?

**Comment by alayek** [comment]
I am not the right person for that. I think perhaps @QuincyLarson or @BerkeleyTrue should take a look and determine this.

**Comment by Rafase282** [comment]
@richyvk I'm closing this PR because of the conflicts it has with regards to other commits that got included. As for the content itself once the allowed links are determined, feel free to open a new PR for it.

**Pull request closed without merging**